src/components/Workbench.js

```diff
@@ -49,15 +50,24 @@ const Workbench = (props) => {
   const calculateRecipe = () => {
     const ingredients = map(selections, (item) => item);
     const potionId = matchRecipes(ingredients);
+    console.log(potionId);
     if (!potionId) {
       alert("Well, I guess you could call this a potion...");
       dispatch(updatePotion(potionId));
     } else {
       alert(`Discovered ${potions[potionId].name}`);
       dispatch(updatePotion(potionId));
+      const clearIngredients = [];
+      dispatch(updateIngredients(clearIngredients));
```

Comment:
this probably needs some planning, right now this will just clear the potion right away and we don't want that -- so we can remove.